src/main/scala/cognite/spark/ThreeDModelRevisionMappingsRelation.scala

```diff
@@ -15,7 +15,7 @@ class ThreeDModelRevisionMappingsRelation(config: RelationConfig, modelId: Long,
 
   override def toRow(t: ThreeDAssetMapping): Row = asRow(t)
 
-  override def uniqueId(a: ThreeDAssetMapping): Long = a.nodeId + a.assetId
+  override def uniqueId(a: ThreeDAssetMapping): String = a.nodeId.toString + a.assetId.toString
```

Comment:
If we decide to keep it as a `Long`: This seems possibly not very unique, we should do `a.nodeId << 32 + a.assetId` or something... which means we may be better off just going to string in this case like you do here.

Reply:
True, true. I kept is as a String, doubt this endpoint will ever hold a lot of results anyways(?).

Reply:
If it's automated it could end up being a lot more, but that may never happen and if it does we'll take care of it then.